core/src/epicli/data/common/ansible/playbooks/roles/rabbitmq/tasks/main.yml

```diff
@@ -5,6 +5,14 @@
 
 - include_tasks: "install-packages-{{ ansible_os_family | lower }}.yml"
 
+# NOTE: rabbitmqctl start_app, stop_app, join_cluster commands do NOT work when rabbitmq-server instance is down
+- name: Ensure rabbitmq-server service is running
+  systemd:
```

Comment:
The `service` module is used in all other places of the role. I would do it the same way.